src/rgw/rgw_lc.h

```diff
@@ -481,6 +481,9 @@ class RGWLC : public DoutPrefixProvider {
     std::mutex lock;
     std::condition_variable cond;
     WorkPool* workpool{nullptr};
+    std::shared_ptr<RGWCoroutinesManager> crs;
```

Comment:
@mattbenjamin @cbodley  @yehudasa .. to avoid creation of http_manager for each object transition, I have included it to be part of worker class. It will be started by the worker thread the first time an object is being transitioned to cloud and stopped post processing all the entries in that bucket. Please let me know if it sounds fine.

Reply:
@soumyakoduri seems reasonable to me.